### PR TITLE
Include wireserver error code in exception message.

### PR DESCRIPTION
**Pull request by vimish**

### CODE REVIEW

**Comment by ejarvi** [comment]
LGTM

**Review by ejarvi — APPROVED** [review]

File: VMEncryption/main/EncryptionSettingsUtil.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
     import http.client as httpclient #python3+
 except ImportError:
     import httplib as httpclient #python2
+
+import xml.etree.ElementTree as ET
```
Comment (vermashi) [inline VMEncryption/main/EncryptionSettingsUtil.py:34]: Do we need to separately install this package or we expect it to always be there?

Comment (vimish) [reply VMEncryption/main/EncryptionSettingsUtil.py:34]: We expect this package to be there. It is part of python standard library

File: VMEncryption/main/EncryptionSettingsUtil.py [hunk 2]
```diff
@@ -297,6 +300,18 @@ def _post_to_wireserver_helper(self, msg_data, http_util):
...
                     raise e
 
+    def get_fault_reason(self, content_xml):
+        try:
+            xml_root = ET.fromstring(content_xml)
+        except:
+            self.logger.log("Exception occured while parsing error xml.")
+            return "Unknown"
+        detail_element = xml_root.find('Details')
+        if detail_element is not None and (detail_element.text is not None and len(detail_element.text) > 0):
```
Comment (vermashi) [inline VMEncryption/main/EncryptionSettingsUtil.py:310]: could be potentially shrunk to:
"detail_element is not None and detail_element.text:"

**Comment by vermashi** [comment]
Please run unit tests against both P2 and P3.

LGTM

**Pull request merged**